articles/libraries/lock/v10/lock-passwordless-migration-guide.md

```diff
@@ -0,0 +1,138 @@
+---
+section: libraries
+toc: true
+description: lock-passwordless to Lock 10 with Passwordless Mode Migration Guide
+---
+# lock-passwordless to Lock 10 with Passwordless Mode Migration Guide
```

Comment:
I might re-word this just because it looks nicer to me when headings start with a capital letter, i.e.

`# Migration Guide for lock-passwordless to Lock 10 with Passwordless Mode` or even just `Migration Guide for lock-passwordless`